src/commands/imageSource/ImageSourceListStep.ts

```diff
@@ -23,14 +24,18 @@ export class ImageSourceListStep extends AzureWizardPromptStep<IImageSourceBaseC
 
         const placeHolder: string = localize('imageBuildSourcePrompt', 'Select an image source for the container app');
         const picks: IAzureQuickPickItem<ImageSourceValues | undefined>[] = [
-            { label: imageSourceLabels[0], data: ImageSource.ContainerRegistry, suppressPersistence: true },
-            { label: imageSourceLabels[2], data: ImageSource.RemoteAcrBuild, suppressPersistence: true },
+            { label: imageSourceLabels[0], data: ImageSource.ContainerRegistry, suppressPersistence: true }
         ];
 
         if (context.showQuickStartImage) {
             picks.unshift({ label: imageSourceLabels[1], data: ImageSource.QuickStartImage, suppressPersistence: true });
         }
 
+        const isVirtualWorkspace = workspace.workspaceFolders && workspace.workspaceFolders.every(f => f.uri.scheme !== 'file');
```

Comment:
Doesn't have to be addressed in this PR, but this is screaming for a utility method in utils to more generically test what sort of situation you're in--desktop, remote from desktop, remote from browser, just browser, etc.